### DISCUSSION TITLE
Welcome to Typed Racket Discussions!

### DISCUSSION THREAD

**Discussion by samth** [racket/typed-racket#1002] [General]
## 👋 Welcome!

  We’re trying out Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Discuss improvements to Typed Racket
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by dvanhorn** [comment]
I ❤️ TR.

**Reply by samth** [reply]
We love you too.

**Comment by pmatos** [comment]
This is a fantastic initiative, thanks @samth .

**Comment by Nyanraltotlapun** [comment]
Hi. Good to have such place.
